e2e/e2e_test.go

```diff
@@ -126,6 +126,38 @@ func TestCheckE2E(t *testing.T) {
 				resource.Version{PR: developPullRequestID, Commit: developCommitID, CommittedDate: developDateTime},
 			},
 		},
+
+		{
+			description: "check works with custom base context",
```

Comment:
Same here. `check` does not use the arguments so these tests are not really testing anything?